sdk/nodejs/provisioning/docker-provision/dockerprovision.ts

```diff
@@ -0,0 +1,6 @@
+// trim image digests to 16 characters to make output more readable
+const digestLen = 16;
+const containerNamePrefix = "dagger-engine";
+const engineSessionBinPrefix = "dagger-engine-session-";
+const containerEngineSessionBinPrefix = "/usr/bin/" + engineSessionBinPrefix;
```

Comment:
Seems like these aren't used yet? Sorry if you're still in the middle of code updates, just commenting in case this is an oversight

Reply:
There's not used and I shall delete them actually, it's already integrated in `image.ts`

Reply:
the file `dockerprovision.ts` has been removed.